app/controllers/edit-account.js

```diff
@@ -0,0 +1,30 @@
+import Ember from 'ember';
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>'Ember' is not defined.<br>'from' is not defined.